package.json

```diff
@@ -12,7 +12,8 @@
 		"format:eslint": "eslint ./src --fix || true",
 		"format:prettier": "prettier . --write",
 		"format:sqlfluff": "./venv/bin/sqlfluff fix src/database/",
-		"lint": "npm run lint:eslint && npm run lint:prettier && npm run lint:tsc && npm run lint:openapi && npm run lint:sqlfluff",
+		"lint": "npm run lint:eslint && npm run lint:prettier && npm run lint:tsc && npm run lint:openapi && npm run lint:sqlfluff && npm run lint:docker-compose",
+		"lint:docker-compose": "dclint ./compose-ci.yml",
```

Comment:
I see there is a `.dclintrc`

If possible, it would be great to configure which files dclint is configured to lint in that location rather than as a CLI parameter.